src/enums/log_event_signature.ts

```diff
@@ -0,0 +1,9 @@
+export enum Log_Event_Signature {
```

Comment:
What are the constants used for?

Reply:
Transfer function's signature?

Reply:
yes, hard-coded transfer function signature.